### ISSUE TITLE
VoiceOver does not read previous instruction when "speak hints" is off

### ISSUE THREAD

**Issue by dsato80** [hulop/NavCog2#31]
Since in VoiceOver environment, "previous instruction", "surround info", and "accessibility info" are set as "hint text" for corresponding button, it does not speak anything if "speak hints" option is off.
